receiver/prometheusreceiver/internal/prom_to_otlp.go

```diff
@@ -0,0 +1,39 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"net"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/translator/conventions"
+)
+
+func createNodeAndResourcePdata(job, instance, scheme string) pdata.Resource {
+	host, port, err := net.SplitHostPort(instance)
+	if err != nil {
+		host = instance
+	}
+	resource := pdata.NewResource()
+	attrs := resource.Attributes()
+	attrs.UpsertString(conventions.AttributeServiceName, job)
```

Comment:
@alolita Does anyone own the spec for prometheus mapping of the standard conventions? These seem somewhat arbitrary. If they can be excluded, I'd only add the prometheus specific attributes for now

Reply:
@anuraaga this is a straight up translation of what already exists in the current converter.

Reply:
I know - it doesn't block this PR but someone needs to be working on that.

Reply:
Cool!

Reply:
Maybe we need to open an issue to track this.

Reply:
> Maybe we need to open an issue to track this.

Why upsert the job to service.name? Shouldn't it be better to check if service.name has been set?